server/src/main/java/org/opensearch/bootstrap/BootstrapChecks.java

```diff
@@ -751,4 +755,25 @@ public BootstrapCheckResult check(BootstrapContext context) {
             );
         }
     }
+
+    /**
+     * Bootstrap check that if a search node contains multiple data paths
+     */
+    static class MultipleDataPathCheck implements BootstrapCheck {
```

Comment:
Can this be private?

Reply:
Why should this class be private? I think it is reasonable to be default

Reply:
The answer to my question is "no" because it is referenced in the unit test, which I missed when I made this comment :)